webrtc/lib/webrtc_endpoint.ex

```diff
@@ -850,6 +842,18 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
     {[], state}
   end
 
+  defp subscribe_tracks(rtc_engine, endpoint_id, new_outbound_tracks) do
+    {valid, invalid} =
+      Enum.reduce(new_outbound_tracks, {[], []}, fn track, {valid, invalid} ->
+        case Engine.subscribe(rtc_engine, endpoint_id, track.id) do
+          {:ok, updated_track} -> {[updated_track | valid], invalid}
+          :ignored -> {valid, [track | invalid]}
+        end
+      end)
+
+    {Enum.reverse(valid), Enum.reverse(invalid)}
```

Comment:
I think the split_with was easier to read. And I'm not sure if order is relevant.